apiAuditLogs/apiAuditLogs.go

```diff
@@ -0,0 +1,277 @@
+		"getSessions":                          true,
+		"getSupportedTimezones":                true,
+		"getSystemPing":                        true,
+		"getTeam":                              true,
+		"getTeamByName":                        true,
+		"getTeamIcon":                          true,
+		"getTeamMember":                        true,
+		"getTeamMembers":                       true,
+		"getTeamMembersByIds":                  true,
+		"getTeamMembersForUser":                true,
+		"getTeamsForScheme":                    true,
+		"getTeamsForUser":                      true,
+		"getTeamStats":                         true,
+		"getTeamsUnreadForUser":                true,
+		"getTeamUnread":                        true,
+		"getTotalUsersStats":                   true,
+		"getUser":                              true,
+		"getUserAccessToken":                   true,
+		"getUserAccessTokens":                  true,
+		"getUserAccessTokensForUser":           true,
+		"getUserByEmail":                       true,
+		"getUserByUsername":                    true,
+		"getUsers":                             true,
+		"getUsersByGroupChannelIds":            true,
+		"getUsersByIds":                        true,
+		"getUsersByNames":                      true,
+		"getUserStatus":                        true,
+		"getUserStatusesByIds":                 true,
+		"getUserTermsOfService":                true,
+		"getWebappPlugins":                     true,
+		"listAutocompleteCommands":             true,
+		"listCommands":                         true,
+		"openDialog":                           true,
+		"patchChannelModerations":              true,
+		"pinPost":                              true,
+		"pushNotificationAck":                  true,
+		"saveReaction":                         true,
+		"searchAllChannels":                    true,
+		"searchArchivedChannelsForTeam":        true,
+		"searchChannelsForTeam":                true,
+		"searchEmojis":                         true,
+		"searchGroupChannels":                  true,
+		"searchPosts":                          true,
+		"searchTeams":                          true,
+		"searchUserAccessTokens":               true,
+		"searchUsers":                          true,
+		"setPostUnread":                        true,
+		"submitDialog":                         true,
+		"teamExists":                           true,
+		"teamMembersMinusGroupMembers":         true,
+		"testElasticsearch":                    true,
+		"testEmail":                            true,
+		"testLdap":                             true,
+		"testS3":                               true,
+		"testSiteURL":                          true,
+		"unpinPost":                            true,
+		"updateUserStatus":                     true,
+		"uploadFileStream":                     true,
+		"viewChannel":                          true,
+	}
+
+	if pass.Pkg.Path() != "github.com/mattermost/mattermost-server/v5/api4" {
```

Comment:
Let's have "github.com/mattermost/mattermost-server/v5" as a constant and concatenate the string with the appropriate suffix wherever needed.

Reply:
done